UnityProject/Assets/Scripts/Objects/Engineering/Reactor/ReactorControlConsole.cs

```diff
@@ -20,6 +55,42 @@ public void SuchControllRodDepth(float requestedDepth)
 			}
 		}
 
+		private void OnDisable()
+		{
+			if (CustomNetworkManager.IsServer == false) return;
+			UpdateManager.Remove(CallbackType.PERIODIC_UPDATE, UpdateMe);
+		}
+
+		private void UpdateMe()
+		{
+			if (ReactorChambers is null) return;
+			if (ReactorChambers.MeltedDown == false || ReactorChambers.PresentNeutrons < 200) return;
```

Comment:
can we perhaps move the check to wherever the numbers are changed instead of periodically asking?

Reply:
I've mentioned this on discord and I guess no one saw what I said. So I'll repeat it again here.

I have no clue how the reactor code works especially with how ugly and all-over the place it is, so I cannot move it anywhere else. Someone else can later improve this so that the reactors themselves are the ones alerting the consoles instead of the console monitoring how they work; but for now I'm leaving this as is.